conformance/utils/suite/suite.go

```diff
@@ -145,8 +145,11 @@ func (suite *ConformanceTestSuite) Setup(t *testing.T) {
 	t.Logf("Test Setup: Ensuring GatewayClass has been accepted")
 	suite.ControllerName = kubernetes.GWCMustBeAccepted(t, suite.Client, suite.TimeoutConfig, suite.GatewayClassName)
 
+	suite.Applier.GatewayClass = suite.GatewayClassName
+	suite.Applier.ControllerName = suite.ControllerName
```

Comment:
Seems like we need a docs change to go along with this if we're going to move forward with this approach.

Reply:
done

Reply:
To clarify, I was trying to say that we need an update to our user-facing docs here: https://github.com/kubernetes-sigs/gateway-api/blob/main/site-src/concepts/conformance.md#running-tests

Reply:
Actually misunderstood this, we're already reading this from GatewayClass resource so users don't need to configure this.